routing/turns_tts_text.cpp

```diff
@@ -54,17 +54,25 @@ std::string GetTtsText::GetTurnNotification(Notification const & notification) c
   {
     if (notification.m_useThenInsteadOfDistance &&
         notification.m_turnDirPedestrian == PedestrianDirection::None)
-      return std::string{};
+      return {};
   }
 
   if (notification.m_useThenInsteadOfDistance && notification.m_turnDir == CarDirection::None)
-    return std::string{};
+    return {};
 
   std::string const dirStr = GetTextById(GetDirectionTextId(notification));
   if (dirStr.empty())
-    return std::string{};
+    return {};
 
-  std::string const distStr = GetTextById(GetDistanceTextId(notification));
+  std::string distStr;
+  if (notification.m_useThenInsteadOfDistance)
+  {
+    distStr = GetTextById("then");
```

Comment:
Should a comma be inserted here before "then"? Like ", then"?

Reply:
The notation is defined for tts, no any commas needed.